modules/data-loading/pages/data-streaming-connector/azure-blob.adoc

```diff
@@ -0,0 +1,153 @@
+= Stream Data from Azure Blob Storage
+:description: Instructions on how to stream data from Azure Blob Storage from the Data Streaming Connector.
+:sectnums:
+
+You can create a data connector between TigerGraph's internal Kafka server and your Azure Blob Storage (ABS) with a specified topic.
+The connector streams data from data sources in your ABS containers to TigerGraph's internal Kafka cluster.
+
+You can then create and run a loading job to load data from Kafka into the graph store using the xref:kafka-loader/index.adoc[Kafka loader].
+
+== Prerequisites
+
+* Your have an link:https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview[Azure storage account] with access to the files you are streaming from.
+
+== Procedure
+
+=== Specify connector configurations
+
+The connector configurations provide the following information:
+
+* Connector class
+* Your Storage account credentials
+* Information on how to parse the source data
+* Mapping between connector and source file
+
+==== Connector class
+
+[.wrap,text]
+----
+connector.class=com.tigergraph.kafka.connect.filesystem.azure.AzureBlobSourceConnector
+----
+
+The connector class indicates what type of connector the configuration file is used to create.
+Connector class is specified by the `connector.class` key.
+For connecting to ABS, its value is
+`connector.class=com.tigergraph.kafka.connect.filesystem.azure.AzureBlobSourceConnector`.
+
+==== Storage account credentials
+
+You have two options when providing credentials:
+
+[tabs]
+====
+Shared key authentication::
++
+--
+Shared key authentication requires you to provide your storage account name and account key.
+
+You can find both the account name and account key on the Access Key tab of your storage account:
+
+.Azure storage account page Access Keys tab
+image::azure-storage-account.png[Azure storage account page Access Keys tab]
+
+To specify the account name and key, use the following configuration.
+Replace `<account_name>` with your account name and replace `<account_key>` with your account key:
+
+[.wrap,text]
+----
+file.reader.settings.fs.azure.account.key.<account_name>.dfs.core.windows.net="<account_key>"
+----
+
+--
+Service principal authentication::
++
+--
+To use service principal authentication, you must first register your TigerGraph instance as an application provide it access to your storage account.
```

Comment:
To use service principal authentication, you must first register your TigerGraph instance as an application and grant it access to your storage account.